### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by Brenntron**
Pagination works. I need to refactor tests to increase speed. Eliza had said to use a constance and mocks but I am not versed in their usage.

### CODE REVIEW

File: spec/views/list_has_20_items_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+feature "List only has twenty items per page" do
+  scenario "posts list has 20 posts per page" do
+    me = Fabricate(:user)
+    30.times do
+      Fabricate(:post, author: me)
+    end
+    signin_as me
+    render_template layout: "users/index"
```
Comment (elizamarcum) [inline spec/views/list_has_20_items_spec.rb:8]: What is this supposed to do?

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class PostsController < ApplicationController
   before_action :require_login, except: [:index, :show]
 
   def index
-    @posts = @user.posts.all
+    @posts = @user.posts.order("created_at").page(params[:page]).per(20)
```
Comment (elizamarcum) [inline app/controllers/posts_controller.rb]: Change `20` to `PER_PAGE`.

Then, create an initializer in config/initializers along the lines of:

```
PER_PAGE = Rails.env.test? ? 5 : 20
```

File: spec/views/list_has_20_items_spec.rb (hunk 1 above)
Comment (Brenntron) [inline spec/views/list_has_20_items_spec.rb:8]: lines 3-6 were to sign in as a user and to make posts attributed to that user. I needed that to test if items in a list. then the render_template layout was to see if I was on the correct page. I was having issues with assert_template and the docs I found said that render would do the same thing.

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (Brenntron) [inline app/controllers/posts_controller.rb]: Done. The file is config/initializers/kaminari.rb. Does the naming matter? I'll change the tests to reflect the changes.

File: spec/views/list_has_20_items_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+feature "List only has twenty items per page" do
+  scenario "posts list has 20 posts per page" do
```
Comment (elizamarcum) [inline spec/views/list_has_20_items_spec.rb]: The only tiny, tiny flaw here is that this still says "20"

Comment (Brenntron) [inline spec/views/list_has_20_items_spec.rb]: Ah. Nice catch. Should be fixed now.

**Pull request merged**